README.md

```diff
@@ -5,3 +5,42 @@
 
 # mumuki-gobstones-server
 > Sinatra server for validating Gobstones tests within [Mumuki](http://github.com/mumuki)
+
+#Prerequisites
+  install rbenv:
+		http://uqbar-wiki.org/index.php?title=Gu%C3%ADa_de_Instalaci%C3%B3n_de_Ruby		 	
+
+
+#Setup
+ Open  the Terminal Console, located in this project's
+ directory, run the following commands: 
+	
+	bundle install
+	
+In order to install repository's submodules:
+	
+	git submodule update --init-- resursive
+
+
+#RUN TESTS
+
+	Note: When running tests you may recieve a Warning like this 		one:
```

Comment:
This looks a little weird, I recommend using back-ticks instead of indentation. See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code